doc/man/8/ceph-bluestore-tool.rst

```diff
@@ -57,6 +57,10 @@ Commands
 
    Instruct BlueFS to check the size of its block devices and, if they have expanded, make use of the additional space.
 
+:command:`bluefs-bdev-emergency-expand` --path *osd path*
```

Comment:
IIUC this command differs from regular bluefs-bdev-expand in flag setting only. Why not to set it unconditionally on any expansion and leave just a single command?